Combine Public & Microsoft Peering in ExR FAQ

ryanbor opened the pull request:
As Public peering is deprecated and has been rolled into Microsoft Peering, we should remove references to it in the FAQ and other docs.

azurecla commented:
Hi __@SimonCropp__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

SimonCropp commented on articles/service-bus/service-bus-azure-and-service-bus-queues-compared-contrasted.md:
```diff
@@ -218,13 +177,9 @@ This section compares the management features provided by Azure Queues and Servi
-- Names of Azure queues can be 3-63 characters long, can contain lowercase letters, numbers, and hyphens. For more information, see [Naming Queues and Metadata](https://msdn.microsoft.com/library/azure/dd179349.aspx).
-
+- Names of Azure queues can be 3-63 characters long, can contain lowercase letters, numbers, and hyphens. For more information, see [Naming Queues and Metadata](https://msdn.microsoft.com/library/azure/dd179349.aspx).
 - Service Bus queue names can be up to 260 characters long and have less restrictive naming rules. Service Bus queue names can contain letters, numbers, periods (.), hyphens (-), and underscores (_).
```
note that the "periods, hyphens and underscores" the characters should be enclosed in single ticks since atm it is invalid md. as u ca see by much of the rest of this PR being in italic

azurecla commented:
__@SimonCropp__, Thanks for signing the contribution license agreement so quickly! Actual humans will now validate the agreement and then evaluate the PR.
<br />Thanks, AZPRBOT;

PRmerger commented:
@SimonCropp : Thanks for your contribution to the Azure documentation! The author, @sethmanheim, has been notified to review your proposed change.

sethmanheim commented:
@SimonCropp Hi, and thank you for your contribution! We think that the extra line makes the bulleted list more readable. That said, this article is scheduled for a major review and update, so we will take your suggestion under consideration at that time. Again, thanks for taking the time!

sethmanheim commented:
@tysonn Please close PR.

SimonCropp commented:
Wanting spacing between li for readability is fine. But that should be done with css so as to not bloat the html and be concistent

sethmanheim commented:
@tysonn OK to close.

SeanFeldman commented:
@sethmanheim perhaps [this comment](https://github.com/Azure/azure-content/pull/6938#discussion_r69509640) should be addressed or at least acknowledged as something that will be done.

sethmanheim commented:
@SeanFeldman I believe I already mentioned that we will take the suggestion under consideration, but for now, we don't use CSS. Thanks again for all the suggestions!

SeanFeldman commented:
@sethmanheim my apology on insisting about this, but @SimonCropp talked about not escaping the special characters that are breaking the rendering, not the CSS aspect.
For example, the following text will cause everything to become in italics because of the special (MD) chacracter `_`.

>  Service Bus queue names can contain letters, numbers, periods (.), hyphens (-), and underscores (_).

That's something that doesn't really require consideration, and just needs to be addressed. Doesn't it? :smiley: 
Thank you for helping.

sethmanheim commented:
@SeanFeldman Not really, because it looks fine on Azure.com. Maybe I'm not understanding the issue? In the meantime I've removed those special characters as they didn't really add anything to the explanation.

SeanFeldman commented:
Great. Thank you @sethmanheim

sethmanheim commented:
@tysonn OK to close.

ryanbor committed:
Combine Public & Microsoft Peering in FAQ

As Public peering is deprecated and has been rolled into Microsoft Peering, we should remove references to it in the FAQ and other docs.
